composer.json

```
@@ -18,7 +18,7 @@
         "friendsofphp/php-cs-fixer": "^3.64",
         "illuminate/container": "^10.48",
         "nette/utils": "^4.0",
-        "sebastian/diff": "^5.1",
+        "sebastian/diff": "^5.1 || ^6.0",
```

just use single version:

```suggestion
        "sebastian/diff": "^6.0",
```

and update phpunit to v11

```
"phpunit/phpunit": "^11.0"
```